Modix.Data/Repositories/MessageRepository.cs

```diff
@@ -176,17 +292,31 @@ from ranked_users
             return stats;
         }
 
-        public async Task<MessageEntity> GetMessage(ulong messageId)
+        /// <inheritdoc />
+        public async Task<MessageBrief> GetMessage(ulong messageId)
         {
-            return await ModixContext.Messages.FindAsync(messageId);
+            return await ModixContext.Messages
+                .AsNoTracking()
+                .AsQueryable()
+                .Select(MessageBrief.FromEntityProjection)
+                .Where(x => x.Id == messageId)
```

Comment:
Does this work? I thought we did

```cs
.AsNoTracking()
.Where(x => x.Id == messageId)
.AsExpandable()
.Select(MessageBrief.FromEntityProjection)
```

Reply:
Yeah - for the purpose of appeasing our EF Linq translation overlords, I'd recommend sticking filters as close to the top of the chain as possible, though I don't know definitively whether it matters here.

Reply:
Looking at it I genuinely don't even know what I was trying to accomplish, fixing asap